ee/codegen/src/__test__/__snapshots__/workflow.test.ts.snap

```diff
@@ -2,9 +2,12 @@
 
 exports[`Workflow > write > should generate correct code when there are input variables 1`] = `
```

Comment:
This test seem to be doing the same thing as the no input variables one right after it, so made the snapshot more consistent with the labelled intent of the test